### DISCUSSION TITLE
System try icon size

### DISCUSSION THREAD

**Discussion by processor286** [BuddiesOfBudgie/budgie-desktop#367] [Q&A] [answered]
Hey everyone. Just getting in to Budgie through EndeavorOS. Like what I see!

When I make the dock icons bigger, the system tray icons do not resize, and it looks really odd, these teeny-tiny icons along side the big menu button and applications, plus they're hard (for me) to see [ this is why I have large dock icons ].

Any way to make these scale up with the rest of the dock? I searched as much as I could , could not find an option for this

edit: same goes for icons in the application launcher, I'd like the ability to scale these up if I can -- not to the extent of the dock though, but larger would be better.

Thanks for any help!

**Comment by JoshStrobl** [comment, marked as answer]
This is already sorted with the new system tray implementation for Budgie 10.8.

**Reply by processor286** [reply]
Thank you, I will wait for that.

**Reply by WanderingOnline** [reply]
Hi Josh

Hope that you don't mind me adding that to me the system tray icons are now too big relative to the Menu icon and the height of the taskbar. For me that has caused Budgie to lose some of it's elegance.
